### PR TITLE
Update README

### LINKED ISSUE #310: calicoctl link is broken
The main page of this repo says:

> If you are looking for the golang version of calicoctl it is in the process of moving and can be found in our calico-containers repo / golang branch.

Hyperlink goes here: https://github.com/projectcalico/calico-containers/tree/golang

However, that link is dead -- presumably this should be https://github.com/projectcalico/calico-containers/tree/master/calicoctl or similar.

### PR DESCRIPTION
**Pull request by gunjan5**
Fixes #310

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
 This repositiory contains Calico's Go components:
 
 - `libcalico`, which can be imported as `"github.com/projectcalico/libcalico-go/lib"`
 
-This library is used by both `calicoctl`, `calico-cni` and `felix`.
-
-If you are looking for the golang version of `calicoctl` it is in the process of moving and
-can be found in our
-[calico-containers repo / golang branch](https://github.com/projectcalico/calico-containers/tree/golang).
+This library is used by [calicoctl](https://github.com/projectcalico/calicoctl), [Calico cni-plugin](https://github.com/projectcalico/calico-cni), [Calico libnetwork-plugin](https://github.com/projectcalico/libnetwork-plugin) and [felix](https://github.com/projectcalico/felix).
```
Comment (robbrockbank) [inline README.md]: cni-plugin -> CNI plugin
libnetwork-plugin -> libnetwork plugin   (don't think we need the hyphen do we?)

Comment (caseydavenport) [inline README.md]: Also the repo names have been changed a bit.

**Comment by gunjan5** [comment]
@caseydavenport I've updated it as per yours and @robbrockbank's suggestions. PTAL

**Comment by caseydavenport** [comment]
LGTM

**Pull request merged**